hardware/ThermostatHardware.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <iostream>
+#include "DomoticzHardware.h"
+
+class CThermostatHardware : public CDomoticzHardwareBase
+{
+public:
+
+	//thermostat available mode in string "OFF,ECO,CONFOR,AUTO," separated by comma, 
+	// and in the order from 0 to N and 0 = OFF 1 = ECO, 2 = CONFOR 3 = AUTO
+	std::string  AvailableMode;
+
+	//thermostat function
+	//return the thermostat available mode in string "OFF,ECO,CONFOR,AUTO," separated by comma , and 0=OFF 1=ECO, 2=CONFOR 3=AUTO
+	virtual std::string GetAvailableMode() { return AvailableMode; };
+	virtual void        SetAvailableMode(std::string  pAvailableMode) { AvailableMode = pAvailableMode; };
```

Comment:
```suggestion
	virtual void SetAvailableMode(std::string pAvailableMode) { AvailableMode = pAvailableMode; };
```
Just curious about why they are not `const std::string&` everywhere?